app/src/main/res/values/strings.xml

```diff
@@ -291,36 +291,36 @@
     <string name="persistent_notification_title">Persistent notification</string>
     <string name="persistent_notification_summary">Add a notification when the database is open</string>
     <string name="advanced_unlock">Advanced unlock</string>
-    <string name="advanced_unlock_explanation_summary">Use advanced unlocking to open a database more easily</string>
+    <string name="advanced_unlock_explanation_summary">Use automated database unlocking</string>
```

Comment:
What does "advanced" do? "Easier" is could do with some improvement.

Reply:
Advanced unlocking is a setting section with tools to unlock.

Reply:
@J-Jamet I didn't understand that, but is it OK, or does it need to include "advanced"?

Reply:
This is a description for education screens, it should be "advanced" and must contain the description in order to know why it is done -> unlock more easily

Reply:
@J-Jamet Advanced doesn't tell the user _why_ it is advanced, whereas "automated" does.

Reply:
Advanced tells me nothing. More easily is a clue, but not an answer to what becomes easier.

Reply:
"Use automated unlocking to open a database more easily" ? If this sentence is accepted, the class names must be changed in the code to be consistent.

Reply:
Sounds good to me.